### ISSUE TITLE
memory issues when backpropgating

### ISSUE THREAD

**Issue by DecentMakeover** [Hsuxu/Loss_ToolBox-PyTorch#5]
Hey 
BinaryFocalTversky loss not able to backpropogate due to memory issues.Any suggestions?

**Comment by Hsuxu** [comment]
It may caused by an inplace operation. Please ref to #8 .
